jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListBasicFiltersPresenter.java

```diff
@@ -175,11 +177,24 @@ protected void addProcessIdFilter() {
                                     f -> addSearchFilter(f, equalsTo(COLUMN_PROCESS_ID, f.getValue())));
     }
 
+    protected void addSLAComplianceFilter() {
+        Map<String, String> aliasMap = new HashMap<String, String>();
+        aliasMap.put("0", constants.SlaNA());
```

Comment:
Please also add options in alphabetical order, makes easier to select in the UI.